ansible_base/authentication/models/authenticator.py

```diff
@@ -5,6 +5,14 @@
 from ansible_base.lib.utils.models import prevent_search
 
 
+def get_next_authenticator_order():
+    """
+    Returns the next authenticator order, which is equals to max(order) + 1
+    """
+    largest_order_authenticator = Authenticator.objects.all().order_by('-order').first()
+    return largest_order_authenticator.order + 1 if largest_order_authenticator else 1
```

Comment:
```suggestion
    largest_order_authenticator = Authenticator.objects.values('order').order_by('-order').first()
    return largest_order_authenticator['order'] + 1 if largest_order_authenticator else 1
```

Small nitpick, but better to just select the field we need when we can.